app/views/requests/show.html.erb

```diff
@@ -57,21 +93,39 @@
 <p>
   <strong>Users with sudo rights:</strong>
   <ul>
-    <% @request.sudo_user_assignments.each do |assignment| %>
-      <p>
-        <li><%= assignment.user.email %></li>
-      </p>
+    <% if current_user.admin? && @request.pending? %>
+      <%= select_tag 'sudo_user_ids',
+                    options_from_collection_for_select(User.all,
+                                                     :id,
+                                                     :email, 
+                                                     @request.sudo_user_assignments.collect {|sudo_user| sudo_user.user_id}),
+                    { multiple: true, class: "selecttwo", style: 'width: 50%' } %>
+    <% else %>
+      <% @request.sudo_user_assignments.each do |assignment| %>
+        <p>
+          <li><%= assignment.user.email %></li>
+        </p>
+    <% end %>
   <% end %>
   </ul>
 </p>
 
 <p>
   <strong>Users:</strong>
   <ul>
-    <% @request.non_sudo_user_assignments.each do |assignment| %>
-        <p>
-          <li><%= assignment.user.email %></li>
-        </p>
+    <% if current_user.admin? && @request.pending? %>
```

Comment:
I also think wwe should completly rework this part, and use a accept button and not a seperate form (I dont know if this works with the controller emthod only)